src/squlearn/util/pennylane/pennylane_circuit.py

```diff
@@ -143,7 +143,7 @@ def __init__(
             self._executor = Executor("pennylane")
 
         # Transpile circuit to supported basis gates and expand blocks automatically
-        self._qiskit_circuit = transpile(circuit, basis_gates=qiskit_pennylane_gate_dict.keys())
+        self._qiskit_circuit = transpile(circuit, basis_gates=qiskit_pennylane_gate_dict.keys(), optimization_level=0)
```

Comment:
Ok, I understand the issue with the `R_Z`-gate getting removed.
BUT: Is this the right approach to fix it? Wouldn't it be better to avoid having the `R_Z` gate in the first place? I'm not saying that we can't do that, but it sounds suboptimal to me.

Reply:
In the discussion with David, we came to the conclusion that at this point the circuit optimization should not be done by qiskit. At another point (in transpiled_encoding_circuit?) a similar problem occurs and there things might be different, so David wanted to look over it how to deal with this. Generally, now i removed the rz gate from the default in circuit measurement qcnn circuit, sothat it will not occur anymore in the default version.

Reply:
In the discussion with David, we came to the conclusion that at this point the circuit optimization should not be done by qiskit. At another point (in transpiled_encoding_circuit?) a similar problem occurs and there things might be different, so David wanted to look over it how to deal with this. Generally, now i removed the rz gate from the default in circuit measurement qcnn circuit, sothat it will not occur anymore in the default version.

Reply:
sounds good 🙂